src/components/editor/mirror-mark/styles.css

```diff
@@ -459,3 +462,25 @@ span.CodeMirror-selectedtext { background: none; }
 	height: 100% !important;
 	overflow: scroll !important;
 }
+
+/* Preview */
+.CodeMirror.CodeMirror-has-preview .CodeMirror-scroll{
```

Comment:
Scroll is broken. If the text is longer than visible area, then scrolling does not work. In both editor and preview mode. If this isn't a simple fix, then this can be a separate issue,

Reply:
Scrolling is broken in general. I will create another issue.